test/transport/transportPolynomials.cpp

```diff
@@ -0,0 +1,87 @@
+#include "gtest/gtest.h"
+
+#include "cantera/transport/TransportFactory.h"
+#include "cantera/thermo/ThermoFactory.h"
+#include "cantera/transport/MixTransport.h"
+
+using namespace Cantera;
+
+class TransportPolynomialsTest : public testing::Test
+{
+public:
+    TransportPolynomialsTest() {
+        phase = newPhase("gri30.cti");
+    }
+
+    void check_viscosity_poly(const std::string speciek, vector_fp visc_coeff_expected, int mode) {
```

Comment:
This is a very minor optimization, but for consistency with what we do generally, non-trivial types should be passed by const reference. I'd also suggest `species_k` as the variable name:
```suggestion
    void check_viscosity_poly(const std::string& species_k, const vector_fp& visc_coeff_expected, int mode) {
```